docs/specification.md

```diff
@@ -1317,6 +1326,42 @@ response.
 The client will get a `originId` field in `RunResult` if the `originId` field in
 the `RunParams` is defined.
 
+### Debug Request
+
+The debug request is sent from the client to the server to debug build target(s). The
+server launches a [Microsoft DAP for Java](https://github.com/microsoft/vscode-java-debug) instance
```

Comment:
We should keep this part language agnostic if possible. So I think it's better to link to the DAP specifiaction rather than to the Java implementation.

```suggestion
server launches a [Microsoft DAP](https://microsoft.github.io/debug-adapter-protocol/) server
```